src/Faithlife.WebRequests/Json/AutoWebServiceResponse.cs

```diff
@@ -10,6 +10,21 @@ namespace Faithlife.WebRequests.Json
 	/// </summary>
 	public abstract class AutoWebServiceResponse
 	{
+		/// <summary>
+		/// Gets the HTTP headers on the response.
+		/// </summary>
+		public HttpResponseHeaders Headers { get; private set; }
```

Comment:
Not 100% sure that making this type part of the public API is the right design decision; @ejball?

Reply:
I would feel more comfortable *allowing* this property on the derived type and setting it if it is there, which is more consistent with the original design.

Reply:
Thanks for the feedback.  How would you feel about that derived type living in this repo?  That would help accomplish my goal of having a common base class that can be used by common utility code that needs access to headers and status codes.

Reply:
At this point I would prefer it lived in a Faithlife-internal NuGet package. `FaithlifeWebClient` or something?